desks-web-app/src/stores/booking-module.tsx

```diff
@@ -94,9 +95,7 @@ const getters: GetterTree<BookingsState, RootState> = {
                 return false
             }
             const key = `${roomId}-${moment(date).format('YYYY-MM-DD')}`
-            const weekKey = `${roomId}-${moment(date).startOf('week').format('YYYY-MM-DD')}`
             return (state.loadingByRoomAndDate.has(key) && state.loadingByRoomAndDate.get(key))
-              || (state.loadingByRoomAndWeek.has(weekKey) && state.loadingByRoomAndWeek.get(weekKey))
```

Comment:
This changes the behaviour of this getter. The old implementation also returned true when the week the day is part of is loading.

I started to clean-up the booking loading code unfortunately that particular part isn't working right now. Otherwise I had suggested we merge my branch und you would be able to use my changes. https://github.com/ChristophBe/desks/tree/feature%2Fhomogenize-booking-loading

Reply:
I have merged the  change so it might need to adjust your code.

Reply:
should be done.    ✅